src/geometry.rs

```diff
@@ -67,7 +67,7 @@ where
     /// This is typically used when computing total padding.
     ///
     /// **NOTE:** this is *not* the width of the rectangle.
-    pub(crate) fn horizontal_axis_sum(&self) -> T {
+    pub(crate) fn horizontal_axis_sum(&self) -> Option<T> {
         self.start + self.end
```

Comment:
How can I leverage the `MaybeMath` trait here?

Reply:
Use .maybe_add